planet/api/client.py

```diff
@@ -270,3 +270,77 @@ def get_assets_by_id(self, item_type, id):
         '''
         url = 'data/v1/item-types/%s/items/%s/assets' % (item_type, id)
         return self._get(url).get_body()
+
+    def get_mosaics(self):
+        '''Get information for all mosaics accessible by the current user.
+
+        :returns: :py:Class:`planet.api.models.Mosaics`
+        '''
+        url = self._url('basemaps/v1/mosaics')
+        return self._get(url, models.Mosaics).get_body()
+
+    def get_mosaic_by_name(self, name):
+        '''Get the API representation of a mosaic by name.
+
+        :param name str: The name of the mosaic
+        :returns: :py:Class:`planet.api.models.Mosaics`
+        :raises planet.api.exceptions.APIException: On API error.
+        '''
+        params = {'name__is': name}
```

Comment:
This api parameter is still undocumented, but I'm not sure there's another way to do this (other than iterating through every item returned and doing client-side filtering).